upload_custom_snippet_ds.py

```diff
@@ -0,0 +1,26 @@
+# from datasets import load_dataset, Dataset
```

Comment:
is this needed?

Reply:
We will need this later. Can we add it in this PR so that I can work on it later?